app/src/docs/v1.api-spec.yaml

```diff
@@ -1152,7 +1152,7 @@ components:
           example: john.smith@example.com
         contactApplicantRelationship:
           type: string
-          enum: [Owner, Employee, Agent, Consultant, Other]
+          enum: [Property owner, Project consultant, Other]
```

Comment:
Data Standards: I'm not too certain on the linkages between these actors, but I would recommend double checking to see if this enum should be changing to these descriptors with adjectives attached, as these terms may be more specific than is necessary.